tests/test_rds2/test_rds2.py

```diff
@@ -556,6 +556,40 @@ def test_create_db_snapshots_copy_tags():
     )
 
 
+@mock_rds2
+def test_copy_db_snapshots():
+    conn = boto3.client("rds", region_name="us-west-2")
+    conn.create_db_snapshot.when.called_with(
```

Comment:
This should be a standalone test, something like `test_copy_db_snapshot_fails_for_unknown_dbinstance`

It should also use pytest.raises, so we can verify that it throws the correct error:
```
with pytest.raises(ClientError) as exc:
    ...
err = exc.value.response["Error"]
err["Message"].should.equal("...?")
```

Reply:
Done